packages/plugin-hardhat/src/index.ts

```diff
@@ -35,6 +37,7 @@ import {
   GetVerifyDeployBuildInfoFunction,
   GetBytecodeDigestFunction,
 } from './defender-v1/verify-deployment';
+import debug from './utils/debug';
```

Comment:
Let's add this import inside the function where it's used:

```ts
const debug = await import('./utils/debug');
```

Originally it was a goal to not have any imports at the top level in this file (other than type imports). I think at some point this wasn't followed properly. We shouldn't have an import of `@upgrades/core` at the top level, and there are some of these import statements that should be written as `import type`.

See https://hardhat.org/hardhat-runner/docs/advanced/building-plugins#optimizing-your-plugin-for-better-startup-time

Reply:
Removed top level imports